### PR TITLE
Add cypress test for pageskins

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Add a test for pageskins, testing if the pageskin image is applied on desktop and that Single Request Mode is used, also check the opposite on tablet/mobile.

Refactored `getTestUrl` slightly, to remove DCR option, we're always on DCR now and in the future.

## Why?
We wan't to be confident that the logic is working correctly, especially single request mode, it's not easy to notice if it's not behaving as we'd like.

Bonus: [Puppies Test Pageskin](https://www.theguardian.com/uk?adtest=puppies-pageskin)

### CODE REVIEW

**Review by emma-imber — APPROVED** [review]

File: cypress/fixtures/pages/liveblogs.ts [hunk 1]
```diff
@@ -7,8 +7,8 @@ const liveblogs: Page[] = [
 	{
 		path: getTestUrl(
 			stage,
-			'/politics/live/2022/jan/31/uk-politics-live-omicron-nhs-workers-coronavirus-vaccines-no-10-sue-gray-report?live=true',
-			{ isDcr: true },
+			'/politics/live/2022/jan/31/uk-politics-live-omicron-nhs-workers-coronavirus-vaccines-no-10-sue-gray-report',
```
Comment (domlander) [inline cypress/fixtures/pages/liveblogs.ts:10]: Removing `?live=true` causes the other Cypress-3 test to fail. I am fixing the other test in my PR, which will fix this one too, so happy for you to leave it

File: cypress/e2e/parallel-4/pageskin.cy.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		it(`Test ${frontWithPageSkin.path} on ${breakpoint.breakpoint} should display the pageskin background and use single request mode`, () => {
+			cy.viewport(breakpoint.width, breakpoint.height);
+
+			cy.intercept(gamUrl, (req) => {
+				req.continue((res) => {
+					expect(res.headers).to.have.property('google-lineitem-id');
+					expect(
+						(res.headers['google-lineitem-id'] as string).split(',')
+							.length,
+					).to.be.greaterThan(1);
```
Comment (domlander) [inline cypress/e2e/parallel-4/pageskin.cy.ts:30]: Does this `expect` test that we're using `single request mode`. If so, why?

**Comment by domlander** [comment]
> we're always on DCR now and in the future

Bold to say we'll always be on DCR in the future 😛

**Comment by domlander** [comment]
Nice refactor and great to have pageskin tests

File: cypress/fixtures/pages/liveblogs.ts (hunk 1 above)
Comment (Jakeii) [reply cypress/fixtures/pages/liveblogs.ts:10]: It's still there just added here now: https://github.com/guardian/commercial/pull/992/files#diff-188823ef01a17b9942962471319b2f75645e1bea6bd18c38eafb078ae0daf56aR29

File: cypress/e2e/parallel-4/pageskin.cy.ts (hunk 2 above)
Comment (Jakeii) [reply cypress/e2e/parallel-4/pageskin.cy.ts:30]: Tom Duffet showed us the `google-lineitem-id` header during the incident, in single request mode it'll have all the ads on the page, in "normal" mode it'll just have a single lineitem.

Will clear it up in the code

File: cypress/fixtures/pages/liveblogs.ts (hunk 1 above)
Comment (Jakeii) [reply cypress/fixtures/pages/liveblogs.ts:10]: Ah the urls weren't quite correct for liveblogs when running locally, fixed.

**Review by domlander — APPROVED** [review]

**Pull request merged**